website/tosti/templates/tosti/base.html

```diff
@@ -35,13 +35,9 @@
     <meta name="theme-color" content="#e8b365">
 </head>
 <body>
-    <nav class="navbar navbar-expand-md sticky-top site-header mb-5">
-        <div class="container-fluid">
-            <a class="navbar-brand" href="/"><img src="{% static 'tosti/svg/TOSTI-logo.svg' %}" height="60"/></a>
-            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarResponsive">
-                <span class="navbar-toggler-icon"></span>
-            </button>
-            <div class="collapse navbar-collapse" id="navbarResponsive">
+    <nav class="navbar navbar-expand-md sticky-top site-header mb-5 navbar-dark">
```

Comment:
I really like this lay-out, it is said it is not this way on mobile.

Reply:
Also rt